### PR TITLE
(Git-3) Use GitCli to Commit Migrations

### PR DESCRIPTION
**Pull request by ytausch**
#### Description:

This is the next step after #2873 and is intended to be merged after.

#### Checklist:

- [x] Pydantic model updated or no update needed

### CODE REVIEW

**Comment by ytausch** [comment]
@beckermr rebase complete!

**Comment by ytausch** [comment]
@jaimergp If you want to review, feel free to do so as well!

File: conda_forge_tick/auto_tick.py [hunk 1]
```diff
@@ -274,19 +308,10 @@ def run(
...
-    feedstock_dir = str(context.local_clone_dir.resolve())
-
-    # This is needed because we want to migrate to the new backend step-by-step
-    repo: github3.repos.Repository | None = github3_client().repository(
-        context.git_repo_owner, context.git_repo_name
-    )
-
-    assert repo is not None
-
+    # feedstock_dir must be absolute
```
Comment (beckermr) [inline conda_forge_tick/auto_tick.py:311]:
```suggestion
    # feedstock_dir must be absolute path
```

**Review by beckermr — CHANGES_REQUESTED** [review]
One typo, but otherwise LGTM!

**Review by beckermr — APPROVED** [review]

**Pull request merged**